src/Microsoft.AspNetCore.Build/NuGetResilientPush.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.Configuration;
+using MSBuildTask = Microsoft.Build.Utilities.Task;
+
+namespace Microsoft.AspNetCore.Build
+{
+    public class NuGetResilientPush : MSBuildTask
+    {
+        private volatile bool _success = true;
```

Comment:
It's set to `false` inside the `Parallel.ForEach` which may be run in multiple threads.